src/MonacoContainer/MonacoContainer.tsx

```diff
@@ -1,13 +1,21 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
 
-import Loading from '../Loading';
-
-import styles from './styles';
+import styles from "./styles";
+import Loading from "../Loading";
 
 // ** forwardref render functions do not support proptypes or defaultprops **
 // one of the reasons why we use a separate prop for passing ref instead of using forwardref
 
+type ContainerProps = {
+  width: number | string;
+  height: number | string;
+  isEditorReady: boolean;
+  loading: React.ReactNode | string;
```

Comment:
could you please import `ReactNode` and `RefObject` from `'react'` in the first line? let's avoid `React.[something]` syntax

Reply:
gotcha - updated 👍🏼